vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -94,6 +97,13 @@ class LocationLiveMapViewFragment @Inject constructor() : VectorBaseFragment<Fra
     private fun setupMap() {
         val mapFragment = getOrCreateSupportMapFragment()
         mapFragment.getMapAsync { mapboxMap ->
+            val bottomSheetHeight = BottomSheetBehavior.from(views.bottomSheet).peekHeight
+            mapboxMap.uiSettings.apply {
+                // Place copyright above the user list bottom sheet
+                setLogoMargins(dimensionConverter.dpToPx(8), 0, 0, bottomSheetHeight + dimensionConverter.dpToPx(8))
+                setAttributionMargins(dimensionConverter.dpToPx(96), 0, 0, bottomSheetHeight + dimensionConverter.dpToPx(8))
```

Comment:
Could you also add this `setAttributionMargins` in `MapTilerMapView`? In this view, we only call `setLogoMargins`.

Reply:
It is working well with default margins. Since I moved the position of attribution here I also needed re-set all the margins.

Reply:
We actually see both logo and the info icon in `MapTilerMapView`. You can see it in maximized static location sharing.

Reply:
I think the info icon is missing in the screen where user selects location sharing option. It is hidden under the options view. This is due to missing call to `setAttributionMargins`.

<img src="https://user-images.githubusercontent.com/46314705/172815961-c6311b38-0102-4221-bb2d-c9e76e4e4802.png" width=300/>

Reply:
Oh, I missed this screen!

Reply:
Nice catch, done!